src/pool.c

```diff
@@ -699,6 +699,27 @@ account_hr(double *avg, const char *address)
     pthread_rwlock_unlock(&rwlock_acc);
 }
 
+void
+account_whr(double *avg, const char *address, const char *rig_id)
```

Comment:
Minor point, but I'd name this worker_hr and move it to after worker_list (and in the header). That way these methods all either are "account_" or "worker_" and neatly grouped. The title should be "stats: worker hashrate" (as "stats: per worker" suggests there are more stats than just hashrate).

Reply:
will rename and move